src/main/java/utilities/DriverFactory.java

```diff
@@ -1,22 +1,22 @@
-package utilities;
-
-import java.util.concurrent.TimeUnit;
-
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-
-public class DriverFactory {
-	public static WebDriver driver;
-	
-	public static void startChromeDriver () {
-		System.setProperty("webdriver.chrome.driver", Constants.CHROME_DRIVER_PATH_MAC);
-		driver = new ChromeDriver();
-		driver.manage().window().maximize();
-		driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
-	}
-
-	public static void stopChromeDriver() {
-		driver.close();
-		driver.quit();
-	}
-}
+package utilities;
+
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class DriverFactory {
+	public static WebDriver driver;
+	
+	public static void startChromeDriver () {
+		System.setProperty("webdriver.chrome.driver", Constants.CHROME_DRIVER_PATH_WINDOWS);

```

Comment:
This is a temporary solution to make it work in your workspace, but will not work for mac, I will be putting in a fix for this. Please discard this change.